### PR TITLE
Post details - main

### PR DESCRIPTION
**Pull request by matej494**

### CODE REVIEW

**Comment by dkulundzic** [comment]
Lets resolve the conflicts before continuing with the review.

File: Marsiple/App/Posts/PostsViewController.swift [hunk 1]
```diff
@@ -11,22 +11,25 @@ import SnapKit
...
+                             failure: { error in
                                 print(error.errorDescription) })
     }
     
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+        
+    override func viewDidAppear(_ animated: Bool) {
+        tabBarController?.tabBar.isHidden = false
```
Comment (dkulundzic) [inline Marsiple/App/Posts/PostsViewController.swift]: This is no longer necessary, since we're using `hidesBottomBarWhenPushed`.

**Review by dkulundzic — DISMISSED** [review]
🚀! We have one minor quibble to be fixed, then feel free to proceed with the merge.

**Comment by matej494** [comment]
Successfully merger branch `post-details/main` into branch `develop` after resolving conflicts.

**Pull request merged**